### DISCUSSION TITLE
Theme Changed in production

### DISCUSSION THREAD

**Discussion by Abdullah-moiz** [ueberdosis/tiptap#5620] [General Questions] [answered]
I am facing an issue , in my localhost my theme for rich texteditor is light when i push the code in production it turn dark 

**Localhost**
![image](https://github.com/user-attachments/assets/7649e4a9-e43b-425d-b400-aa7ec205c83c)

**Production** 
![image](https://github.com/user-attachments/assets/48caf57a-3d95-483c-9d16-a9d1e7d874f1)

**Comment by nperez0111** [comment, marked as answer]
This is squarely your application. Tiptap is headless and does not provide dark theme styling perhaps you have a dark mode extension installed. This is not to do with tiptap

**Comment by KhalilMeziane** [comment]
You can resolve the issue by forcing the theme using the dark={false} prop on the RichTextEditor component.
`<RichTextEditor
	{...props}
	dark={false}
/>`

In cases where the editor theme doesn't align with the app theme, manually set the dark prop using dark={theme === 'dark'}
